examples/tree_1d_dgsem/elixir_shallowwater_twolayer_dam_break.jl

```diff
@@ -0,0 +1,100 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Semidiscretization of the two-layer shallow water equations for a dam break test with a 
+# discontinuous bottom topography function to test entropy conservation
+
+equations = ShallowWaterTwoLayerEquations1D(gravity_constant=9.81, H0=2.0, rho_upper=0.9, rho_lower=1.0)
+# This initial condition will be overwritten with the discontinuous initial_condition_dam_break
+initial_condition = initial_condition_convergence_test
```

Comment:
Why is this then set as such in the first place? As an outsider, I found it confusing and I suspect other students might feel the same way...

Reply:
Could you maybe add this reason (or an abbreviated version of it) to the elixir such that it is clearly motivated why this is necessary without having to read the rest of the elixir code?

Reply:
Sure, I have added a comment in 4f2114ca248344248107f39fd2d7e0523083b079